### PR TITLE
input-field: add `accept` support to `type=file` inputs

### PR DESCRIPTION
**Pull request by stephenmathieson**

### CODE REVIEW

File: src/input-field/index.js [hunk 1]
```diff
@@ -159,6 +162,11 @@ export function render({ props, state }, setState) {
     onBlur, onFocus
   };
 
+  // add accept support to <input type=file>s
+  if (type === 'file' && accept) {
```
Comment (dominicbarnes) [inline src/input-field/index.js]: is it really important that `type === 'file'`? I've generally just attached whatever attributes the user passes through. (ie: they can stick "min" and "max" even if they didn't pick a `number` type)

File: src/input-field/index.js [hunk 2]
```diff
@@ -134,7 +137,7 @@ export function afterMount({ props }, el) {
  */
 export function render({ props, state }, setState) {
   // general props
-  let { autofocus, disabled, name, placeholder, readonly, size, type, value } = props;
+  let { autofocus, disabled, name, placeholder, readonly, size, type, value, accept } = props;
```
Comment (dominicbarnes) [inline src/input-field/index.js]: can you put this in alphabetical order?

File: src/input-field/index.js (hunk 1 above)
Comment (stephenmathieson) [inline src/input-field/index.js]: nope, not important. just figured it made sense.

will fix.

File: src/input-field/index.js [hunk 3]
```diff
@@ -95,7 +95,10 @@ export let propTypes = {
   validationMessage: { type: 'function' },
 
   // Sets the input value.
-  value: { type: 'string' }
+  value: { type: 'string' },
+
+  // Adds accept to type=file inputs.
+  accept: { type: 'string' }
```
Comment (dominicbarnes) [inline src/input-field/index.js]: can you move this up higher? (to be in alphabetical order)

**Pull request merged**